### PR TITLE
Skeleton attempt at a new python wrapper package.

### PR DESCRIPTION
**Pull request by ewels**
A bit messy and raw, but hopefully provides a starting point for something cool 🎉

### CODE REVIEW

File: wrapper/Sarek.egg-info/PKG-INFO [hunk 1]
```diff
@@ -0,0 +1,45 @@
+Metadata-Version: 1.0
+Name: Sarek
+Version: 0.1.dev0
+Summary: Helper tools for use with the Sarek Nextflow pipeline.
+Home-page: https://github.com/SciLifeLab/Sarek
+Author: Maxime Garcia, Szilvester Juhos, Phil Ewels
+Author-email: maxime.garcia@scilifelab.se, phil.ewels@scilifelab.se
+License: MIT License
+
+Copyright (c) 2016 SciLifeLab
```
Comment (marcelm) [inline wrapper/Sarek.egg-info/PKG-INFO]: It’s 2018 :-)

**Comment by marcelm** [comment]
You added the `.pyc` files to the commit.

File: wrapper/Sarek.egg-info/PKG-INFO (hunk 1 above)
Comment (ewels) [reply wrapper/Sarek.egg-info/PKG-INFO]: Oops, those files shouldn't even be in there..!

**Comment by ewels** [comment]
* face palm *

**Comment by ewels** [comment]
Removed all of the compiled python files and updated `.gitignore` to avoid this happening again. Rebased those commits to keep things tidy. Thanks @marcelm!

File: wrapper/Sarek.egg-info/PKG-INFO (hunk 1 above)
Comment (ewels) [reply wrapper/Sarek.egg-info/PKG-INFO]: Ok, licence file also updated to 2018 ;)

party

**Pull request merged**